testing/gearmand/APKBUILD

```diff
@@ -0,0 +1,75 @@
+# Contributor: Aaron Hurt <ahurt@ena.com>
+# Maintainer: Aaron Hurt <ahurt@ena.com>
+pkgname=gearmand
+pkgver="1.1.12"
+pkgrel=2
+pkgdesc="Gearman job server and client"
+url="http://gearman.org"
+arch="all"
+license="BSD"
+depends="libgearman"
+depends_dev="boost-dev gperf libevent-dev util-linux-dev mariadb-dev
+	hiredis-dev openssl-dev sqlite-dev libmemcached-dev"
+makedepends="$depends_dev"
+install="$pkgname.pre-install $pkgname.post-deinstall"
+subpackages="$pkgname-doc libgearman libgearman-dev"
```

Comment:
The main purpose is the daemon ```gearmand``` but the libs are needed for the SDKs for PHP and other languages.  The website itself is *gearman.org* and the title is just *Gearman* so I think ```gearman-libs```, ```gearman-dev``` and a binary ```gearmand`` provided by the main package sound reasonable.